code/obj/item/clothing/helmets.dm

```diff
@@ -492,7 +492,7 @@
 	name = "welding helmet"
 	desc = "A head-mounted face cover designed to protect the wearer completely from space-arc eye. Can be flipped up for clearer vision."
 	icon_state = "welding"
-	c_flags = SPACEWEAR | COVERSEYES
+	c_flags = SPACEWEAR | COVERSEYES | UNCHOKE
```

Comment:
should probably only block choking when flipped down

Reply:
ah i missed that